clients/go/gengo.go

```diff
@@ -381,10 +381,15 @@ func (c *WagClient) do%sRequest(ctx context.Context, req *http.Request, headers
 	}
 	resp, err := c.requestDoer.Do(client, req)
 	if err != nil {
+		c.logger.ErrorD("client-request-finished", map[string]string{
```

Comment:
@kvigen here is an updated attempt: https://github.com/Clever/wag/pull/160/commits/205cd47ba9b2f625e2ff943f3f7c7a4154365db7